Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM alpine:3.11.3
 WORKDIR /opt/bin
 
 #Add Packages
-RUN apk add --no-cache bash sed grep curl coreutils nodejs npm
+RUN apk add --no-cache bash sed grep curl coreutils nodejs npm python3
```

Comment:
Alternative here instead of adding to the Dockerfile for all workers is we split this out into its own worker...